packages/data-models/skin.model.ts

```diff
@@ -0,0 +1,8 @@
+import { IAppConfigOverride } from ".";
```

Comment:
nit(blocking)
I know vscode autocompletes this way, but sometimes I have compiler issues when importing from the index (assuming this file will also be exported from it). Probably safer to put `./constants` (and maybe consider renaming the file to `appConfig` in line with the other changes)